client/src/components/addBook/AddBook.js

```diff
@@ -26,21 +67,23 @@ return(
 
 
 <div class="container">
-<form> 
+<form action ='/' onSubmit={handleSubmit}>  
```

Comment:
So, once the request is submitted, it directs to the homepage?

Reply:
Maybe direct it to the library page?

Reply:
I just put it to homepage for now since we didn't have a page for isLogged  in.